woocommerce/woocommerce-square
# Replace the use of deprecated hook `wcs_renewal_order_meta` with `wc_subscriptions_renewal_order_data`

### All Submissions:



---

### Changes proposed in this Pull Request:
PR makes a minor change to replace the deprecated hook `wcs_renewal_order_meta` with `wc_subscriptions_renewal_order_data`

Closes #179

### Steps to test the changes in this Pull Request:
Detailed steps are given in the issue #179 description, Please verify that no deprecated notice is there in the error logs.

### Changelog entry
> Dev - Replace the use of deprecated hook `wcs_renewal_order_meta` with `wc_subscriptions_renewal_order_data`.
